frontend/src/metabase/static-viz/components/StaticChart/StaticChart.tsx

```diff
@@ -0,0 +1,22 @@
+import type { RawSeries, VisualizationSettings } from "metabase-types/api";
+
+const Placeholder = ({ text }: { text: string }) => {
+  return <div style={{ width: "300px", height: "200px" }}>{text}</div>;
+};
+
+export interface StaticChartProps {
+  rawSeries: RawSeries;
+  dashcardSettings: VisualizationSettings;
+  renderingContext: RenderingContext;
+}
```

Comment:
I belive every static chart component would need that type, right? Would it make sense to pull it up? I was thinking of `metabase/static-viz/types` or maybe `metabase/visualization/types`?

Reply:
Also, it seems like we're missing a proper type for `RenderingContext`?

Reply:
Good catch, thanks! Fixed in the latest commit